src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -189,4 +189,12 @@ public function deleteAction(Request $request, Post $post)
 
         return $this->redirectToRoute('admin_post_index');
     }
+
+    /**
+     * Must be protected to allow Symfony DIC to create a proxy.
+     */
+    protected function getSlugger()
```

Comment:
I would not advertise this feature in the demo (it's for advanced users). Using constructor injection is OK here and works out of the box.
It's also easier to understand for newcomers.